packages/block-editor/src/style.scss

```diff
@@ -56,3 +56,65 @@
 @import "./components/editor-skeleton/style.scss";
 @import "./components/inserter/style.scss";
 
+
+// Default styles for alignments in case the theme doesn't provide them.
+.wp-block {
```

Comment:
Ideally, this should be something that a theme author can copy/paste to its stylesheet and tweak the numbers.
There are some tweaks necessary to achieve it though.

- We should replace the `data-align` with the classNames.
- We should see whether we can remove the `wp-block` className in favor of something closer to what themes do. Something more like `.entry-content > *`

When we'll be able to remove IE11 support, we could use CSS variables for the widths and margins.

Reply:
> * We should see whether we can remove the `wp-block` className in favor of something closer to what themes do. Something more like `.entry-content > *`

If we're refactoring now anyway, I really think this is something that we should do now. I see the alignment wrapper as context to a block or blocks, not as a block itself.

Reply:
Also, if this is the main reason for magically adding the alignment wrapper, and we're going to remove the `wp-block` class anyway, does it still make sense to magically add it? Tbh, I'd much rather have an explicit component for it instead of magically adding wrappers

Reply:
My original ideal in https://github.com/WordPress/gutenberg/issues/20650#issuecomment-595245765 was to have something like:

```jsx
<AlignmentWrapper align={align}>
  <Block.figure />
</AlignmentWrapper>
```

Both in the edit and save function.